drivers/interrupt_controller/intc_ite_it8xxx2.c

```diff
@@ -206,6 +206,25 @@ uint8_t get_irq(void *arg)
 	} while (intc_irq != IVECT);
 	/* determine interrupt number */
 	intc_irq -= IVECT_OFFSET_WITH_IRQ;
+	/*
+	 * Look for pending interrupt if there's interrupt number 0 from
+	 * the AIVECT register.
+	 */
+	if (!intc_irq) {
+		uint8_t int_pending;
+
+		for (int i = (IT8XXX2_IER_COUNT - 1); i >= 0; i--) {
+			int_pending = (*reg_status[i] & *reg_enable[i]);
+			if (int_pending) {
+				intc_irq = (MAX_REGISR_IRQ_NUM * i) +
+						find_msb_set(int_pending) - 1;
+				printk("Pending interrupt found: %d\n",
+						intc_irq);
+				printk("CPU mepc: 0x%lx\n", csr_read(mepc));
```

Comment:
Can we use logging instead of `printk`? This seems more like a `LOG_DBG`

Reply:
Done